src/app/hooks/useResetUserFlow.ts

```diff
@@ -14,6 +14,7 @@ const userFlowConfig: Record<string, { resetTo: string }> = {
   '/confirm-btc-tx': { resetTo: '/send-btc' },
   '/send-brc20': { resetTo: '/account-list' },
   '/confirm-inscription-request': { resetTo: '/account-list' },
+  '/confirm-brc20-tx': { resetTo: '/account-list' },
```

Comment:
`/account-list` is a strange place to reset to. Should it not rather be `/` or `/send-brc20`?

Reply:
there was a fix here: https://github.com/secretkeylabs/xverse-web-extension/pull/538

so I guess it could be '/', but not '/send-brc20' because that route needs some query params to not crash